statsmodels/regression/linear_model.py

```diff
@@ -818,21 +822,42 @@ def iterative_fit(self, maxiter=3):
         ----------
         maxiter : integer, optional
             the number of iterations
+        rtol : float, optional
+            Relative tolerance between estimated coefficients to stop the
+            estimation.  Stops if
+
+            max(abs(last - current) / abs(last)) < rtol
+
         """
-        #TODO: update this after going through example.
-        for i in range(maxiter-1):
+        # TODO: update this after going through example.
+        for i in range(maxiter - 1):
             if hasattr(self, 'pinv_wexog'):
                 del self.pinv_wexog
             self.initialize()
             results = self.fit()
+            self.history.append(results)
```

Comment:
I think we should not store a results instance, since it will be stale right away (point to the wrong model attributes), and might use up memory.
As in GLM and RLM we should store a few attributes that we might use for convergence criteria or as fit information. History is a dict in both models 
GEE has a _fit_history which is also a dict.